### PR TITLE
Add typescript example

### PR DESCRIPTION
**Pull request by stoivo**

### CODE REVIEW

**Comment by stoivo** [comment]
Do you want to include a typescript example?

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ dist/index.js
 example/dist/
 example/package-lock.json
 
+/.pnpm-debug.log
```
Comment (EMH333) [inline .gitignore:12]: Please remove references to pnpm

File: example-js/package.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "main": "buildscript.js",
+  "scripts": {
+    "build": "node buildscript.js"
```
Comment (EMH333) [inline example-js/package.json:4]: Thanks for fixing this!

**Review by EMH333 — COMMENTED** [review]
A great start! Just a few minor things to clean up, but certainly helpful I think!

Please make sure the lint passes as well

File: example-js/buildscript.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ esbuild
         mainFields: ["svelte", "browser", "module", "main"],
         outdir: "./dist",
         format: "esm",
+        logLevel: "info",
```
[the remark this answers is not in the document]
Comment (stoivo) [reply example-js/buildscript.js:17]: Just to have some output when esbuilds runs. So I know it was run

**Comment by stoivo** [comment]
I will update other comments tomottow

File: example-js/buildscript.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (stoivo) [reply example-js/buildscript.js:17]:
```
⋊> ~/D/e/example-ts on 965c329 ⨯ pnpm build                                                                                                                                                               08:31:22

> @ build /Users/simon/Downloads/esbuild-svelte/example-ts
> node buildscript.js


  dist/entry.js   119.9kb
  dist/entry.css    727b 

⚡ Done in 827ms
⋊> ~/D/e/example-ts on 965c329 ⨯ pnpm build                                                                                                                                                               08:32:17

> @ build /Users/simon/Downloads/esbuild-svelte/example-ts
> node buildscript.js

⋊> ~/D/e/example-ts on 965c329 ⨯    
```

File: .gitignore (hunk 1 above)
Comment (stoivo) [reply .gitignore:12]: Is it okay to keep it in .gitignore so it won't be committed by accident?

**Comment by stoivo** [comment]
Updated

**Comment by stoivo** [comment]
When will to have time to look at this?

**Comment by EMH333** [comment]
I just started a new job, I'll be delayed for a few days while I settle in :)

**Comment by stoivo** [comment]
Congrats with new job. No problem.

**Comment by stoivo** [comment]
👋

**Pull request merged**